src/pages/workspace/withPolicy.tsx

```diff
@@ -5,13 +5,17 @@ import type {ComponentType, ForwardedRef, RefAttributes} from 'react';
 import React, {forwardRef} from 'react';
 import type {OnyxEntry} from 'react-native-onyx';
 import {withOnyx} from 'react-native-onyx';
+import type {SettingsNavigatorParamList} from '@navigation/types';
 import policyMemberPropType from '@pages/policyMemberPropType';
 import * as Policy from '@userActions/Policy';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
 import type * as OnyxTypes from '@src/types/onyx';
 
-type PolicyRoute = RouteProp<{params: {policyID: string}}>;
+type PolicyRouteName = typeof SCREENS.WORKSPACE.BILLS | typeof SCREENS.WORKSPACE.INVOICES;
+
+type PolicyRoute = RouteProp<SettingsNavigatorParamList, PolicyRouteName>;
```

Comment:
```suggestion
type PolicyRoute = RouteProp<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.BILLS | typeof SCREENS.WORKSPACE.INVOICES>;
```

NAB: You can keep them inline if you want